### PR TITLE
add docker support

### PR DESCRIPTION
**Pull request by vlccek**
Hi i've made some changes for supporting docker. Now you can build a docker container by 'docker build . ' in the root of repository. You can also use my docker-compose file for inspiration on how to use it. I hope this will be usefull

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    && pip install Pillow \
+    && apk del build-deps
+
+COPY ./requirements.txt requirements.txt
+COPY ./livestream_saver.py livestream_saver.py
+COPY ./livestream_saver livestream_saver
+RUN python3 -m pip install --upgrade pip
+RUN python3 -m pip install -r requirements.txt
+
+COPY ./livestream_saver.cfg.template ./config.yml
```
Comment (glubsy) [inline Dockerfile:18]: Shouldn't this be the following instead?
```Dockerfile
COPY ./livestream_saver.cfg.template ./livestream_saver.cfg
```

Comment (vlccek) [reply Dockerfile:18]: I'm sorry I have an examination at university. I found more bugs in my code. Please close this PR and next week I create a new one with more improvements and fixes.

But yes you are right.

Comment (glubsy) [reply Dockerfile:18]: It's alright. You can push changes to that same branch and it will be updated. There is no rush.

**Comment by vlccek** [comment]
Hi I've made some changes, now should work correctly. Unfortunately, i don't have any experience with uploading (building)images to docker hub.

**Comment by glubsy** [comment]
Thanks!  Looks interestin. I'll toy around with it a bit and merge once I find some free time.

**Comment by glubsy** [comment]
Added a Dockerfile in https://github.com/glubsy/livestream_saver/pull/79
Closing this as this is conflicting with the main branch now.

**Pull request closed without merging**